.circleci/config.yml

```diff
@@ -29,6 +30,10 @@ jobs:
           paths:
             - $HOME/.go_workspace/src/github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/test/vendor
 
+      - run: pip install pre-commit
```

Comment:
Can you add a comment that says something like, "Fail the build if the pre-commit hooks had not been run."

Reply:
Done